lib/constants.ts

```diff
@@ -67,6 +74,10 @@ export const MULTISIG_FEE_COLLECTOR = EthAddressSchema.parse(
   '0x9A142B38d483d150dB2c115b4efA5ca37aC57Ebc'
 )
 
+export const BONDING_SWAP_CONTRACT = EthAddressSchema.parse(
+  '0x0000000000000000000000000000000000000000'
```

Comment:
Did you test the smart contract with the SDK? you should deploy it and see if the SDK works with it

Reply:
also rename to SWAPPER_CONTRACT